### DISCUSSION TITLE
hoax address balance allocation

### DISCUSSION THREAD

**Discussion by 0xmuchen** [Cyfrin/foundry-full-course-cu#1865] [General]
Hello guys, I am currently on my 2 module of the Foundry 101 course and I learning basics of unit testing so my questions in next:
In the code provided by patrick we use SEND_VALUE twice in our which freaks me out a little bit, cause I can't understand is okay not to specify the balance for such situation or not 
(in the docs there a info that default value for now specified one is: 2^128 wei) basically infinite amount of eth.

So I tried just using the hoax(address(i)); and it worked
`function testWithdrawByRealOwnerMultipleFunders() public funded {
        // arrange
        uint160 numberOfFunders = 10;
        uint160 startingFunderIndex = 1;
        // in order to use ints with adresses type casting we need to use int160
        for (uint160 i = startingFunderIndex; i < numberOfFunders; i++) {
            hoax(address(i)), SEND_VALUE;
            // simulate the thnx from the given tnx and eth amount as the balance as seconds argument
            // if no balance for hoax addr specified it will default to the 2^128 wei (huge amount of eth)
            fundMe.fund{value: SEND_VALUE}();

**Comment by EngrPips** [comment]
Hello @dfjlh, I don't understand your question. Can you please clarify this so that I can help clarify your doubts?

**Comment by 0xmuchen** [comment]
@EngrPips  sensei sorry for long response, I somehow fixed it thanks

**Reply by EngrPips** [reply]
Awesome. I am glad you fixed it. Keep crushing it.
